### ISSUE TITLE
module not found

### ISSUE THREAD

**Issue by sudeepisstupid123** [spyder-ide/spyder#14889]
### What steps will reproduce the problem?

from bs4 import BeautifulSoup
in spyder it gives an error of ModuleNotFoundError: No module named 'bs4' 
i have installed using pip install bs4 and it still gives and error i can use the same code in idle but not in spyder

## Versions

* Spyder version: 4.2.3 
* Python version: 3.7.9
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree                    :  None (OK)
IPython >=7.6.0                 :  7.21.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.3.0 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.8.0 (OK)
pylint >=1.0                    :  2.7.2 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.2 (OK)
qtconsole >=5.0.1               :  5.0.2 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.4 (OK)
setuptools >=39.0.0             :  54.0.0 (OK)
sphinx >=0.6.6                  :  3.5.1 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog                        :  1.0.2 (OK)
zmq >=17                        :  22.0.3 (OK)

# Optional:
cython >=0.21                   :  0.29.22 (OK)
matplotlib >=2.0.0              :  3.3.4 (OK)
numpy >=1.7                     :  1.19.3 (OK)
pandas >=1.1.1                  :  1.2.3 (OK)
scipy >=0.17.0                  :  1.6.1 (OK)
sympy >=0.7.3                   :  1.7.1 (OK)
```

**Comment by steff456** [comment]
Hi @sudeepisstupid123,

Are you checking that you are trying to import the package in the correct environment?

**Comment by sudeepisstupid123** [comment]
the module is working in IDLE but not in spyder 
I installed the module using pip and still the module is not be found when I run it in spyder but works in idle

**Comment by sudeepisstupid123** [comment]
> Hey @sudeepisstupid123, quick question: did you use our Windows installer to install Spyder?

I installed using anaconda

**Comment by steff456** [comment]
And you are running spyder from the same environment you have installed BeautifulSoup?

**Comment by sudeepisstupid123** [comment]
@steff456  no i have installed BeautifulSoup in pip and running spyder using anaconda

**Comment by kdevnoobs** [comment]
Hi. I deleted a module that has been working fine for over a year the other day to reinstall an upgraded version of it. So I ended up using pip uninstall modulename then using python setup.py install from the directory with the upgraded file. However, now I get an ModuleNotFoundError: No module named 'modulename ' when using my scripts that worked perfectly and found the module before I re-installed modulename earlier on. The error disappears when I copy the modulename folder in the same file where I have my script. I ran the cmd prompt as administrator. When I try to install the module again it tell me that the requirement is already satisfied, so the module did install. I'm still on spyder 3.7 since I find it more practical than the newer version. Is there a way to refresh the module see if spyder has access to it?
